usr/src/uts/common/fs/zfs/spa.c

```diff
@@ -6653,6 +6566,9 @@ spa_sync(spa_t *spa, uint64_t txg)
 		ddt_sync(spa, txg);
 		dsl_scan_sync(dp, tx);
 
+		if (spa->spa_vdev_removal != NULL)
+			svr_sync(spa, tx);
+
 		while (vd = txg_list_remove(&spa->spa_vdev_txg_list, txg))
```

Comment:
please update it to avoid parentnes gcc warning to:
while ((vd = txg_list_remove(&spa->spa_vdev_txg_list, txg)) != NULL)